ckan/templates/home/robots.txt

```diff
@@ -0,0 +1,26 @@
+<style>
```

Comment:
Why is the robots file being styled?

Reply:
https://www.facebook.com/robots.txt / https://instagram.com/robots.txt / https://www.google.com/robots.txt / https://twitter.com/robots.txt / https://www.linkedin.com/robots.txt / https://www.udemy.com/robots.txt / https://www.youtube.com/robots.txt / https://open.spotify.com/robots.txt / https://www.netflix.com/robots.txt / https://www.hbo.com/robots.txt / https://www.uber.com/gr/en/robots.txt etc....

all are styled in the same way, it's like a pattern

Reply:
@ThrawnCA